packages/manager/modules/pci/src/projects/project/private-networks/private-networks.service.js

```diff
@@ -45,6 +46,33 @@ export default class {
       });
   }
 
+  getLocalPrivateNetworks(serviceName) {
+    return this.$http
+      .get(`/cloud/project/${serviceName}/aggregated/network`)
+      .then(({ data }) => {
+        const privateNetworks = [];
+        data.resources.forEach((network) => {
+          if (network.visibility === 'private' && !network.vlanId) {
+            this.getSubnets(serviceName, network.region, network.id).then(
+              ({ cidr, dhcpEnabled, id, allocationPools }) => {
+                const allocatedIp = allocationPools
+                  .map((ipPool) => `${ipPool.start} - ${ipPool.end}`)
+                  .join(' ,');
+                privateNetworks.push({
+                  cidr,
+                  dhcpEnabled,
+                  id,
+                  allocatedIp,
+                  ...network,
+                });
+              },
+            );
+          }
+        });
+        return privateNetworks;
+      });
```

Comment:
**question:** Can we ask API team to add filter to distinguish between Local and Global private network. Also, please ask API team to add an indicator to determine the type of private network. We cannot rely on `vlan` as it is not explicit enough. In future, what if we have 3rd type of private network and how do you identify it?

Reply:
Yup I will check with API team, if not I can check based on region.